src/solver/main.jl

```diff
@@ -256,10 +262,12 @@ implemented in the function `get_roots` and its children.
 
 - dropmultiplicity (optional): Print repeated roots or not?
 
+- strict (optional): Bool that enables/disables strict assert if input expression is a univariate polynomial or not. If strict=true and expression is not a polynomial, `solve_univar` throws an assertion error.
+
 # Examples
 
 """
-function solve_univar(expression, x; dropmultiplicity=true)
+function solve_univar(expression, x; dropmultiplicity=true, strict=true)
```

Comment:
Should solve_univar be renamed to solve_univar_poly ?

Reply:
`solve_multivar` should be renamed to `solve_multivar_poly` aswell then - i'll do it when i get the time